src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double area() {
+        return side * side;
+    }
+
+    @Override
```

Comment:
The method name 'draw' is fine, but make sure that it is consistent with the interface it's implementing. If the interface method is 'draw', then this is correct.